nsidc-data-cookbook/how-to-guides/netcdf_cf.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "How do I get the bounding box of a NetCDF file in latitude and longitude?"
+author: Andrew P. Barrett
+date: last_modified
+---
+
+## Problem
+My data are in a NetCDF file but there are no latitude and longitude coordinates.  How do I find the bounding box in latitude and longitude?.
+
+## Solution
+### Using xarray and rioxarray
+```
+import rioxarray
+import xarray as xr
+
+
+# Load dataset using decode_coords='all'
+ds = xr.open_dataset('example_data/NSIDC0081_SEAICE_PS_N25km_20230627_v2.0.nc',
+                     decode_coords='all')
+
+# Get the bounds of the data grid in WGS84 (EPSG code 4362)
+ds.rio.transform_bounds(4326)
+```
+```
+(-180.0, 30.98056405144958, 180.0, 90.0)
+```
+
+## Discussion
+
+As of [CF v1.8](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.10/cf-conventions.html#grid-mappings-and-projections), latitude and longitude coordinate variables are no longer required in CF-compliant NetCDF files, as long as projected horizontal spatial coordinates (e.g. `x` and `y`, or `easting` and `northing`) and a `grid_mapping` variable is provided.  The `grid_mapping` variable defines the coordinate reference system (CRS) of the projected horizontal coordinates.
+
+The easiest, and frankly best, way to read and work with NetCDF files is to use `xarray`.  `rioxarray` extends the `xarray` package to use CRS and make geospatial tasks, such as reprojecting and regridding, easier.  By setting the keyword `decode-coords='all'`, `rioxarray` searches the `xarray.DataArray` or `xarray.Dataset` for the CRS.  `rioxarray` also determines the `transform` or `geotransform`, which defines the image CRS that transforms cell coordiates (column and row) into projected coordinates (x, y).  The `transform` is calculated from the coordinates of the data.  See the `rioxarray` [documentaton](https://corteva.github.io/rioxarray/stable/getting_started/crs_management.html#) for more details.
```

Comment:
This is a great paragraph that is also relevant for the lat/lon array how-to. Maybe we could consider moving or copying this to the higher level "work with netcdf files" section if we want to provide a preamble there in a future iteration?

Reply:
That's a good idea.  Let's leave this here for the moment because these how-tos are stand alone at the moment.  We can reorganize later.